skate-plugin/src/main/kotlin/com/slack/sgp/intellij/util/StringExtensions.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright (C) 2023 Slack Technologies, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.slack.sgp.intellij.util
+
+private val pattern = "_[a-z0-9]".toRegex()
+
+fun String.snakeToCamelCase(): String {
```

Comment:
I think we have something like this in SGP, I think in a future PR we should consider moving this to kotlin-cli-util maybe for reuse